libvast/src/chunk.cpp

```diff
@@ -49,6 +53,9 @@ chunk::make(const void* data, size_type size, deleter_type&& deleter) noexcept {
 }
 
 chunk_ptr chunk::make(view_type view, deleter_type&& deleter) noexcept {
+  auto data = view.data();
+  auto sz = view.size();
+  VAST_SDT(chunk_make, data, sz);
```

Comment:
Doesn't it make more sense to do this in the actual constructor?

Reply:
Good point